dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -88,6 +88,7 @@ elif [ "$CONFIG_TYPE" == "unified" ]; then
     CFGGEN_PARAMS=" \
         -d \
         -y /etc/sonic/constants.yml \
+        -T /usr/local/sonic/frrcfgd \
```

Comment:
This might break if anyone uses "unified" config type for bgpcfgd. Did you try making the necessary changes in https://github.com/sonic-net/sonic-buildimage/blob/master/dockers/docker-fpm-frr/frr/gen_frr.conf.j2#L2?

Reply:
This is the first issue listed in https://github.com/sonic-net/sonic-buildimage/issues/2001.

Without this -T, unified mode would not be able to generatee frr.conf. I explained the cause in the issue.

Reply:
-T provides an additional path from original path. 

a.k.a both 
/usr/local/sonic/frrcfgd
and 
/usr/share/sonic/templates

In j2 template env.

https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-config-engine/sonic-cfggen#L463

Reply:
I would suggest going with new config_type for better control. It may be unmanageble when unified is overloaded for multiple use-cases.

Reply:
This change is to fix existing unified mode issue. Currently frr.conf would not be able to generated in unfied mode.

Reply:
As long as this change doesnt break anything, for example, bgpcfgd operation, I'm good.

Reply:
Yes. There is no issue. These paths just provide j2's include file's searching path. 
Since gen_frr.conf.j2 uses two template files, we need to include both of them.